tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "target": "ES2017",
-        "module": "CommonJS",
+        "module": "NodeNext",
```

Comment:
That's the most significant change - it is required for esm, "type": "module", etc, and the main consequence is having to add ".js" to all the imports.